service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/ServiceAuthenticationWithSharedAccessPolicyToken.java

```diff
@@ -47,46 +47,9 @@ public IotHubConnectionString populate(IotHubConnectionString iotHubConnectionSt
      * @param policyName The policy name string
      * @param token The token string
      */
-    public ServiceAuthenticationWithSharedAccessPolicyToken(String policyName, String token)
+    ServiceAuthenticationWithSharedAccessPolicyToken(String policyName, String token)
```

Comment:
private?

Reply:
This constructor is called from another class within this package, so I can't mark it as private. This lack of modifier makes the constructor "Package private" which means only classes within this package can access it. Classes outside of the package, even if they extend this class, can't access it